### PR TITLE
Bugfix for bison parser

### PR DESCRIPTION
**Pull request by dwightguth**

### CODE REVIEW

**Comment by ehildenb** [comment]
@dwightguth is there a reasonable regression test you can include for this bugfix?

**Comment by radumereuta** [comment]
Is this supposed to fix an issue? Or is it just something you noticed while you were working on something else?
Looks good otherwise.

File: k-distribution/tests/regression-new/bison-glr-bug/Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+test: test.kore
+	cat test.kore | diff - test.ref
+
+test.kore: forwarder.iele kompile
+	./iele-kompiled/parser_PGM forwarder.iele > test.kore
+include ../../../include/kframework/ktest.mak
+
+clean:
+	rm -rf test.kore test-kompiled
```
Comment (radumereuta) [inline k-distribution/tests/regression-new/bison-glr-bug/Makefile]: This is missing some file patterns.
I've updated the list in my warnings PR:
`rm -rf $(KOMPILED_DIR) .depend-tmp .depend .krun-* .kprove-* kore-exec.tar.gz`
Is it possible to delete test.kore and still call the command from ktest.mak?

**Review by radumereuta — CHANGES_REQUESTED** [review]

File: k-distribution/tests/regression-new/bison-glr-bug/Makefile (hunk 1 above)
Comment (dwightguth) [reply k-distribution/tests/regression-new/bison-glr-bug/Makefile]: You're welcome to suggest how to do that, but I don't really know myself.

**Review by radumereuta — APPROVED** [review]

**Pull request merged**